docs/readme.md

```diff
@@ -1,6 +1,6 @@
 # Data API builder for Azure Databases
 
-Data API builder for Azure Databases provide modern REST and GraphQL endpoints to your Azure Databases.
+Data API builder for Azure Databases provides modern REST and GraphQL endpoints to your Azure Databases.
 
 With data API builder, database objects can be exposed via REST or GraphQL endpoints so that your data can be accessed using modern techniques on any platform, any language, and any device. With an integrated and flexible policy engine, native support for common behavior like pagination, filtering, projection and sorting, the creation of CRUD backend services can be done in minutes instead of hours or days, giving developers an efficiency boost like never seen before.
```

Comment:
```suggestion
With Data API builder, database objects can be exposed via REST or GraphQL endpoints so that your data can be accessed using modern techniques on any platform, any language, and any device. With an integrated and flexible policy engine, native support for common behavior like pagination, filtering, projection and sorting, the creation of CRUD backend services can be done in minutes instead of hours or days, giving developers an efficiency boost like never seen before.
```